### ISSUE TITLE
State of EasyList Czech and Slovak

### ISSUE THREAD

**Issue by monzta** [tomasko126/easylistczechandslovak#405]
Hi there, 

I was wondering about the state of this filter list. I tried to reach out to Tomas Taro earlier with no response.

There have been several issue reports and pull requests in repo that no one has taken care of until now. 

In order to keep users happy and ad filtering quality high in Adblock Plus and AdBlock, we are considering dropping the recommended status of the filter list and possibly replacing it with a new one. That means that we would no longer offer it as a default filter list for Czech and Slovak users. 

Do you think you can watch the issues section and the pull requests more carefully? Or is there a better way to collaborate with you? 

Thanks!

cc @addisr

**Comment by monzta** [comment]
Tomas got back to me. Closing this.

**Comment by monzta** [comment]
I'm reopening this issue since Tomas isn't responding to my emails.

**Comment by mlnrDev** [comment]
> I'm reopening this issue since Tomas isn't responding to my emails.

Hi, what's the issue? This repo is active again, but I've intentionally skipped PRs and issues that are old and have been thinking about closing them as stale, prompting the author to reopen the PR/issue if it's still happening.

**Comment by monzta** [comment]
@mlnrDev Thanks a lot! Really appreciate you are working on this! It's mainly about the PRs and reported issues. Some of them were reported by team mates from ABP/eyeo and there hasn't been any reaction for months.

**Comment by mlnrDev** [comment]
@monzta as I said, I'll probably close the issues and PRs in question, since most of them are well over a year old and ask the users to reopen if it's still a problem.

**Comment by monzta** [comment]
Fair enough. I'm also looking at the long-term strategy of this list. @tomasko126 signalled interest in migrating the ownership to someone else but I don't know who could take care of that and right now it's difficult to reach him.

**Comment by monzta** [comment]
@mlnrDev Can you share any estimation when you go through the open PRs and issues?

**Comment by mlnrDev** [comment]
> @mlnrDev Can you share any estimation when you go through the open PRs and issues?

hi, I've already gone through old PRs and closed them, I just need to take a look at the issues some time today since they probably need more attention and some of them are still active
